main/acle.md

```diff
@@ -10856,6 +10870,48 @@ Replacing `_hor` with `_ver` gives the associated vertical forms.
     __arm_streaming __arm_inout("za");
 ```
 
+#### BFTMOPA, FTMOPA, STMOPA, UTMOPA
+
+``` c
+  // Only if __ARM_FEATURE_SME_TMOP != 0
+  // Variants are also available for:
+  //   _za16[_f16_f16] (only if __ARM_FEATURE_SME_F16F16 != 0)
+  //   _za16[_bf16_bf16] (only if __ARM_FEATURE_SME_B16B16 != 0)
+  //   _za32[_bf16_bf16]
+  //   _za32[_f16_f16]
+  //   _za32[_s16_s16]
+  //   _za32[_u16_u16]
+  //   _za32[_s8_s8]
+  //   _za32[_u8_u8]
+  void svtmopa_lane_za32[_f32_f32](uint64_t tile, svfloat32x2_t zn, svfloat32_t zm, 
+                              svuint8_t zk, uint64_t imm_idx)
+    __arm_streaming __arm_inout("za");
+
+  // Only if __ARM_FEATURE_SME_TMOP != 0 && __ARM_FEATURE_SME_F8F16 != 0
+  void svtmopa_lane_za16[_f8_f8](uint64_t tile, svfloat32x2_t zn, svfloat32_t zm, 
```

Comment:
Same comments as the FEAT_SME_MOP4,
s/[_f8_f8]/[_mf8_mf8]/
missing _fpm at the end of the intrinsic